### PR TITLE
Condor glitch collection

### PR DESCRIPTION
**Pull request by EthanMarx**
Condorizes collection of glitch strain given omicron trigger files

### CODE REVIEW

**Comment by EthanMarx** [comment]
@wbenoit26 Need to patch up the tests, but should be good for a look over. I ran a short experiment and completed end to end. Running on the full dataset now as well.

File: projects/sandbox/datagen/datagen/scripts/glitches.py [hunk 1]
```diff
@@ -4,121 +4,240 @@
...
-    return glitches, snrs, gpstimes
+            continue
+
+        try:
+            glitch_ts = data.crop(beg, end)
+        except ValueError:
+            logging.warning(f"Data not available for trigger at time: {time}")
+            continue
+        else:
+            print(len(glitch_ts.value))
```
Comment (wbenoit26) [inline projects/sandbox/datagen/datagen/scripts/glitches.py:135]: Print statement

File: projects/sandbox/datagen/datagen/scripts/glitches.py [hunk 2]
```diff
@@ -4,121 +4,240 @@
...
+    pad: Tuple[float, float],
+    snr_thresh: float,
+    sample_rate: float,
+    channel: str,
+    ifo: str,
+    outdir: Path,
+    condordir: Path,
+    accounting_group: str,
+    accounting_group_user: str,
+    request_memory: float = 32678,
```
Comment (wbenoit26) [inline projects/sandbox/datagen/datagen/scripts/glitches.py:161]: Should be 32768 for a power of 2, not that it actually makes a difference

**Review by wbenoit26 — APPROVED** [review]
Two minor nitpicks, looks good to me

File: projects/sandbox/datagen/datagen/scripts/glitches.py (hunk 1 above)
Comment (EthanMarx) [reply projects/sandbox/datagen/datagen/scripts/glitches.py:135]: classic

File: projects/sandbox/datagen/datagen/scripts/glitches.py (hunk 2 above)
Comment (EthanMarx) [reply projects/sandbox/datagen/datagen/scripts/glitches.py:161]: Good eye

**Pull request merged**